client/components/settings/AlertItem.js

```diff
@@ -34,49 +34,51 @@ export default class AlertItem extends React.Component {
     }
 
     render() {
-        let {account, alert} = this.props;
+        let { account, alert } = this.props;
 
         assert(alert.order === 'gt' || alert.order === 'lt');
 
-        return <tr>
-            <td>{account.title}</td>
-            <td>
-                <div className="form-inline">
-                    <span>{this.props.sendIfText}&nbsp;</span>
+        return (
+            <tr>
+                <td>{ account.title }</td>
+                <td>
+                    <div className="form-inline">
+                        <span>{ this.props.sendIfText }&nbsp;</span>
 
-                    <select className="form-control"
-                      defaultValue={alert.order}
-                      ref="selector"
-                      onChange={this.onSelectChange}
-                    >
-                        <option value="gt">{$t('client.settings.emails.greater_than')}</option>
-                        <option value="lt">{$t('client.settings.emails.less_than')}</option>
-                    </select>
+                        <select className="form-control"
+                          defaultValue={ alert.order }
+                          ref="selector"
+                          onChange={ this.onSelectChange }>
+                            <option value="gt">
+                                { $t('client.settings.emails.greater_than') }
+                            </option>
+                            <option value="lt">{ $t('client.settings.emails.less_than') }</option>
```

Comment:
nit: symmetry between the two options, please: either both one-liners, or both on 3 lines